app/static/src/app/wodin.ts

```diff
@@ -14,18 +14,32 @@ import { storeOptions as stochasticStoreOptions } from "./store/stochastic/stoch
 import { initialiseRouter } from "./router";
 import tooltip from "./directives/tooltip";
 import { AppType } from "./store/appState/state";
+import { translate } from "../../translationPackage";
 
 declare let appType: AppType;
 
 const { Basic, Fit, Stochastic } = AppType;
-const getStore = () => {
+const getStoreAndTranslate = () => {
```

Comment:
OK, I'm totally ill-equipped to review this one, so will leave this with @EmmaLRussell I'm afraid.

The only thing that jumps out at me is that `getXandY` seems like an anti-pattern - I presume this comes from/works in hint though?

Reply:
no worries! so this is actually a wodin exclusive! the reason I had to do it like this was because of the way we getStore, the store type could be either `basicStore`, `fitStore` or `stochasticStore` and the `translate(store)` function was complaining about the different types, maybe I can modify the type definitions for this function but seemed easy to just put it in the store function, will take a look at trying to fix it now!

Reply:
okay should be better now!